### PR TITLE
Not add unsubscribed people to referent managed users

### PR DESCRIPTION
**Pull request by illusionOfParadise**

### CODE REVIEW

File: job_referent_managed_users.go [hunk 1]
```diff
@@ -103,5 +104,19 @@ func ReferentManagedUsers() {
 		panic(err)
 	}
 
+	// Removing unsubscribed people
+	ReferentManagedUsersPrint("Removing unsubscribed people")
+	if _, err := db.Exec(`
+        DELETE FROM projection_referent_managed_users
```
Comment (tgalopin) [inline job_referent_managed_users.go]: The identation seems weird, isn't it?

Comment (illusionOfParadise) [reply job_referent_managed_users.go]: I don't know why Github displays it strangely, but in the code it's good.

Comment (tgalopin) [reply job_referent_managed_users.go]: In Go you should use tab for identation, that's why it's displayed differently in Github. Can you change that :) ?

Comment (illusionOfParadise) [reply job_referent_managed_users.go]: Is it better ? ;)

**Pull request merged**